src/test/run-make/llvm-module-pass/Makefile

```diff
@@ -0,0 +1,11 @@
+-include ../tools.mk
+
+all: $(call NATIVE_STATICLIB,llvm-pass)
+	$(RUSTC) plugin.rs -C prefer-dynamic
+	$(RUSTC) main.irs
+
+$(TMPDIR)/libllvm-pass.a: $(TMPDIR)/llvm-pass.o
+	$(AR) rcs $(TMPDIR)/libllvm-pass.a $(TMPDIR)/llvm-pass.o
```

Comment:
You should be able to omit this as `NATIVE_STATICLIB` above it should handle assembling this on platforms like MSVC where `lib.exe` is used.